cat-home/src/test/java/com/dianping/cat/report/task/TaskHelperTest.java

```diff
@@ -22,23 +22,35 @@
 import java.util.Date;
 
 import junit.framework.Assert;
+
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 
+@RunWith(PowerMockRunner.class)
 public class TaskHelperTest {
 
+	@Rule
+	public final ExpectedException thrown = ExpectedException.none();
+
 	@Before
 	public void setUp() throws Exception {
 	}
 
 	@After
 	public void tearDown() throws Exception {
-	}
```

Comment:
This change replaces an 8-space tab with 4 spaces. I think it should be left as it is.